src/sentry/static/sentry/app/views/settings/project/projectSettingsNavigation.jsx

```diff
@@ -1,46 +0,0 @@
-import React from 'react';
-
-import HookStore from 'app/stores/hookStore';
-import SentryTypes from 'app/sentryTypes';
-import SettingsNavigation from 'app/views/settings/components/settingsNavigation';
-import getConfiguration from 'app/views/settings/project/navigationConfiguration';
-import withOrganization from 'app/utils/withOrganization';
-import withProject from 'app/utils/withProject';
-
-class ProjectSettingsNavigation extends React.Component {
-  static propTypes = {
-    organization: SentryTypes.Organization,
-    project: SentryTypes.Project,
-  };
-
-  constructor(props) {
-    super(props);
-    // Allow injection via getsentry et all
-    const org = this.props.organization;
-    const hooks = [];
-    HookStore.get('project:settings-sidebar').forEach(cb => {
```

Comment:
Are we removing this hook?

Reply:
It was totally unused, not typed, and not documented. I can add back if we want

Reply:
idk if we're committed to keeping these around for on prem/custom installs or not - I'm +1 removing it

Reply:
Ah that's a good point.... If someone complains we can bring it back I suppose 🤔 though I know how it feels to have an OSS project just remove some code from under your feet.